cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/WorkflowHttpHandlerTest.java

```diff
@@ -522,6 +522,9 @@ public void testWorkflowForkApp() throws Exception {
     // Stop the program while in fork
     stopProgram(programId, 200);
 
+    // Wait till the program stop
+    waitState(programId, "STOPPED");
```

Comment:
is there a way to use ``ProgramRunStatus`` in ``waitState``? Not for this PR though. If its possible, can you add a note in https://issues.cask.co/browse/CDAP-2984